daiquiri/query/utils.py

```diff
@@ -194,9 +194,22 @@ def get_job_column(job, display_column_name):
     except (ValueError, KeyError):
         return {}
 
-    if schema_name == settings.TAP_UPLOAD:
+    if (schema_name == settings.TAP_UPLOAD) or (schema_name == get_user_schema_name(job.owner)):
```

Comment:
Is that needed anymore, I thought if the metadata comes from the VO table, via the user columns, then it is not needed to fetch it from the database. But maybe I am missing something.

Reply:
That was actually the issue! The metadata were lost when querying user tables. Explaining why the PyVO upload was problematic whereas the Web Upload not. (Since the PyVO upload requires a query on user tables).

Reply:
I see you meant the fetch_column not the line above. indeed for the user_tables fecthing the database, may not be necessary. but for the upload yes. I could add a switch here.